### PR TITLE
Ported Link Button Demo to Vala

### PR DESCRIPTION
**Pull request by urffsamhunt**
Ported the Link Button demo to Vala.

### CODE REVIEW

File: src/Link Button/main.vala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
```
Comment (Diego-Ivan) [inline src/Link Button/main.vala:1]:
```suggestion
#! /usr/bin/env -S vala workbench.vala --pkg gtk4
```

File: src/Link Button/main.vala [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+public void main () {
+    var linkbutton = (Gtk.LinkButton) workbench.builder.get_object ("linkbutton");
+    linkbutton.clicked.connect (() => message ("The link has been visited"));
```
Comment (Diego-Ivan) [inline src/Link Button/main.vala:5]: The signal here must be `notify["visited"]` instead of `clicked`, as in the JS demo.

**Review by Diego-Ivan — CHANGES_REQUESTED** [review]

**Review by Hofer-Julian — APPROVED** [review]
I've adapted the Rust part slightly.
At least the Rust part is good to merge from my side.

File: src/Link Button/main.vala (hunk 2 above)
Comment (Diego-Ivan) [reply src/Link Button/main.vala:5]: @urffsamhunt Thanks for fixing this in the Rust demo! Please make the same changes in Vala

**Review by Diego-Ivan — CHANGES_REQUESTED** [review]

**Pull request merged**